pytensor/link/pytorch/dispatch/extra_ops.py

```diff
@@ -21,3 +21,44 @@ def cumop(x):
             return torch.cumprod(x, dim=dim)
 
     return cumop
+
+
+@pytorch_funcify.register(Repeat)
+def pytorch_funcify_Repeat(op, **kwargs):
+    axis = op.axis
+
+    def repeat(x, repeats, axis=axis):
+        return x.repeat_interleave(repeats, dim=axis)
```

Comment:
Same here
```suggestion
        return x.repeat_interleave(repeats)
```

Reply:
Are you sure about the suggestion? I understand your comments above, but shouldn't the call to the `torch` function "know" about the axis/dim?

Reply:
What do you mean about `knowing`? These variables are constant and not supposed to change, so it's fine to rely on the local scope of when the function is created. We do that all over the place, and seems both more readable (less code duplication) and clear about what are the runtime parameters vs static graph parameters that come from the Op itself.

Reply:
In other words when in pytensor you write sum(x, axis=2), you are defining a sum operation that will always happen at axis=2. It is not allowed to change at runtime, and so it doesn't make sense to "offer that flexibility" on the dispatch. 

Similarly, we could implement `square(x)` in pytorch as `def power(x, pow=2): torch.pow(x, pow)` but it wouldn't make sense, because square can only be a power of 2. `def square(x): torch.pow(x, 2)` would be fine

Reply:
To be clear you still need to provide dims, in your inner function. My suggestion was not a real suggestion, just highlighting the part of the code I was talking about.

Reply:
Makes sense. Please have a look at the current version  and let me know whether I missed something and it still requires changes.

Reply:
Yeah sorry Id didn't get it at first. We want to pass them of course